app/domain/authentication/authn_jwt/identity_providers/identity_from_decoded_token_provider.rb

```diff
@@ -9,7 +9,7 @@ def initialize(authentication_parameters)
           @secret_fetcher = Conjur::FetchRequiredSecrets.new
           @resource_class = ::Resource
           @authentication_parameters = authentication_parameters
-          @resource_id = @authentication_parameters.authn_jwt_variable_id
+          @variable_id = @authentication_parameters.authn_jwt_variable_id
           @decoded_token = @authentication_parameters.decoded_token
```

Comment:
these 2 can go in private methods instead of setting them in the constructor as you set `@authentication_parameters`. just add these
```
def variable_id
  @authentication_parameters.authn_jwt_variable_id
end

def decoded_token
  @authentication_parameters.decoded_token
end
```

You can also you `Forwardable` and `def_delegators` to get the same effect.

Reply:
Done